--- controlplane.yaml	2023-04-19 20:34:26.493063995 +0200
+++ controlplane.yaml.MODIFIED	2023-04-19 20:36:40.424347769 +0200
@@ -53,7 +53,15 @@
         #         - '!10.0.0.3/32'
         #         - fdc7::/16
     # Provides machine specific network configuration options.
-    network: {}
+    network:
+      interfaces:
+        - deviceSelector:
+            busPath: "0*"
+          dhcp: true
+          vip:
+            ip: VIRTUAL_IP
+
+    # network: {}
     # # `interfaces` is used to define the network interface configuration.
     # interfaces:
     #     - interface: enp0s1 # The interface name.
